src/ring_theory/valuation/basic.lean

```diff
@@ -321,6 +321,8 @@ begin
        ... ≤ v (a + s)      : aux (a + s) (-s) (by rwa ←ideal.neg_mem_iff at h)
 end
 
+local attribute [-instance] classical.DLO
```

Comment:
Could you add a comment (linking [to Zulip](https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.233733.20algebraic.20closure)?) explaining why this instance is problematic here?

Reply:
done